src/cs/QuicSimpleTest/Program.cs

```diff
@@ -0,0 +1,4 @@
+﻿using System.Net.Quic;
+
+Console.WriteLine($"QuicConnection.IsSupported = {QuicConnection.IsSupported}");
```

Comment:
Any idea how best to eliminate these warnings?

Reply:
There is an easy way to do it, I'll do it.